internal/sdk/cloudian/sdk_test.go

```diff
@@ -1,16 +1,62 @@
 package cloudian
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"math/rand"
+	"net/http"
+	"net/http/httptest"
 	"reflect"
 	"strings"
 	"testing"
 	"testing/quick"
 )
 
+func TestGetGroup(t *testing.T) {
+	expected := Group{
+		GroupID: "QA",
+	}
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{
+			"groupId": "QA"
+			}`))
+	}))
+	defer mockServer.Close()
+
+	mockClient := &http.Client{}
+
+	req, err := http.NewRequestWithContext(context.TODO(), http.MethodGet, mockServer.URL, nil)
+	if err != nil {
+		t.Fatalf("Failed to create request: %v", err)
+	}
+
+	resp, err := mockClient.Do(req)
+	if err != nil {
+		t.Fatalf("Failed to send request: %v", err)
+	}
```

Comment:
Is the request just to ser that test server respons? Could we skip it and just confirm that it works when part of the sdk call?

Reply:
You are right - these lines can be removed and the test still passes
done